collect_app/src/main/java/org/odk/collect/android/widgets/MediaWidget.java

```diff
@@ -1,15 +1,47 @@
 package org.odk.collect.android.widgets;
 
 import android.content.Context;
+import android.graphics.Color;
 import android.media.MediaPlayer;
 import android.support.annotation.CallSuper;
 
 import org.javarosa.form.api.FormEntryPrompt;
 
+import timber.log.Timber;
+
 public abstract class MediaWidget extends QuestionWidget {
 
+    private int playColor;
+
     public MediaWidget(Context context, FormEntryPrompt prompt) {
         super(context, prompt);
+
+        String imageURI = this instanceof SelectImageMapWidget ? null : prompt.getImageText();
+        String audioURI = prompt.getAudioText();
+        String videoURI = prompt.getSpecialFormQuestionText("video");
+
+        // shown when image is clicked
+        String bigImageURI = prompt.getSpecialFormQuestionText("big-image");
+
+        getQuestionMediaLayout().setAVT(audioURI, imageURI, videoURI, bigImageURI, player);
+        getQuestionMediaLayout().setAudioListener(this);
+
+        initPlayColor();
+    }
+
+    private void initPlayColor() {
+        playColor = themeUtils.getAccentColor();
+
+        String playColorString = getFormEntryPrompt().getFormElement().getAdditionalAttribute(null, "playColor");
+        if (playColorString != null) {
+            try {
+                playColor = Color.parseColor(playColorString);
+            } catch (IllegalArgumentException e) {
+                Timber.e(e, "Argument %s is incorrect", playColorString);
```

Comment:
`Argument %s to Color.parseColor is incorrect` might be helpful.

Reply:
I see you didn’t create, but just moved this code, so disregard the above if you like.